Libs/Core/ctkJobScheduler.cpp

```diff
@@ -291,7 +307,7 @@ void ctkJobScheduler::stopAllJobs(bool stopPersistentJobs)
       }
 
     job->setStatus(ctkAbstractJob::JobStatus::Stopped);
```

Comment:
At this point, the job is not stopped. Why do we set its status to stopped?

It is also possible that we request cancel but the job actually completes successfully (because by the time the worker realizes that cancel was requested, the job is already done).

Reply:
the worker has not beed started yet and the job is still in the main thead (see comment just few lines before). There is no worry that it will be started in meanwhile, because only jobs with status Initialized will started by the scheduler (that's why we put already as status stopped! because effettivelty the job has been stopped!).

Reply:
Please add this explanation as comment in the code

Reply:
ok., done!